src/test/java/com/google/firebase/auth/internal/FirebaseTokenFactoryTest.java

```diff
@@ -18,6 +18,7 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
```

Comment:
Can we also have some unit tests in `FirebaseTokenVerifierImplTest`.

Reply:
Done.